arbitrator/config/arbitrator_params.yaml

```diff
@@ -26,4 +26,4 @@ use_fixed_costs: true
 # Map: The priorities associated with each plugin during the planning 
 # process, values will be normalized at runtime and inverted into costs
 # Unit: N/a
-plugin_priorities: {SCIStrategicPlugin: 3.0, RouteFollowingPlugin: 2.0, WorkZonePlugin: 4.0, PlatooningStrategicPlugin: 1.0}
+plugin_priorities: {SCIStrategicPlugin: 4.0, RouteFollowingPlugin: 2.0, WorkZonePlugin: 4.0, PlatooningStrategicPlugin: 1.0}
```

Comment:
Maybe its not causing an issue but SCIStrategicPlugin and WorkZonePlugin both have 4.0. Maybe setting SCIStrategicPlugin to 5.0 to ensure priority would be preferable?

Reply:
I think since the geofence and spat msgs are not received workzone isn't activated. Workzone wasn't causing any problems, but SCI weight was close to RouteFollowing, that's why I increased it.